crates/composer/src/main.rs

```diff
@@ -14,8 +14,9 @@ fn main() -> Result<()> {
 
     let (_stream, stream_handle) = OutputStream::try_default()?;
 
+    let jukebox = Jukebox::new().context("loading records into jukebox")?;
```

Comment:
nit: The message passed here doesn't reflect the local context - we're not loading any records into the jukebox, we're merely creating a jukebox instance. Whether that implies loading of records is an implementation detail of the jukebox. It may as well load the records at the time they're first played.

I.e. this particular message could be added within the Jukebox `impl`. Here, I think, something like this would fit better

```suggestion
    let jukebox = Jukebox::new().context("creating jukebox")?;
```

Reply:
Right, nicely spotted "break of information encapsulation". Applying suggestion in a fixup commit.